src/sentry/middleware/integrations/parsers/vercel.py

```diff
@@ -0,0 +1,16 @@
+from __future__ import annotations
+
+import logging
+
+from sentry.middleware.integrations.parsers.base import BaseRequestParser
+from sentry.models.outbox import WebhookProviderIdentifier
+
+logger = logging.getLogger(__name__)
+
+
+class VercelRequestParser(BaseRequestParser):
```

Comment:
it'll be cool to have at least one test 😅

Reply:
Added a test!